ibis/expr/operations/core.py

```diff
@@ -1,21 +1,29 @@
 from __future__ import annotations
 
 from abc import abstractmethod
+from typing import Sequence
 
 from public import public
 
 import ibis.expr.rules as rlz
 from ibis.common.exceptions import ExpressionError
 from ibis.common.grounds import Annotable, Comparable
 from ibis.expr.rules import Shape
-from ibis.expr.schema import Schema
-from ibis.util import UnnamedMarker, is_iterable
+from ibis.util import UnnamedMarker
 
 
 @public
 class Node(Annotable, Comparable):
+    @property
+    def args(self):
```

Comment:
I'd be up to removing of `args` and `argnames` in a follow-up.

Reply:
Let's leave this as is for now.